spec/openapi.yaml

```diff
@@ -128,64 +52,40 @@ paths:
                 type: object
                 properties:
                   current:
-                    $ref: "#/components/schemas/OctoConnCurrent"
+                    $ref: "#/components/schemas/ConnCurrent"
                   options:
-                    $ref: "#/components/schemas/OctoConnOptions"
+                    $ref: "#/components/schemas/ConnOptions"
         403:
           $ref: "#/components/responses/Unauthorized"
     post:
-      tags:
-        - octoprint
-      summary: Mock for issue a connection command. Do not do anything.
-      security:
-        - ApiKeyAuth: []
+      summary: Issue a connection command
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                command:
+                  type: string
+                  default: connect
+                baudrate:
+                  type: integer
+                  default: 115200
+                port:
+                  type: string
+                  default: VIRTUAL
+                printerProfile:
+                  type: string
+                  default: _default
       responses:
         204:
           description: No Content
         403:
           $ref: "#/components/responses/Unauthorized"
-  /api/timelapse:
-    get:
-      tags:
-        - octoprint
-      security:
-        - ApiKeyAuth: []
-      summary: Retrieves a moked list of timelapses and the current config.
-      responses:
-        200:
-          description: OK
-          content:
-            application/json:
-              schema:
-                type: object
-                properties:
-                  config:
-                    type: object
-                    properties:
-                      type:
-                        type: string
-                  enabled:
-                    type: boolean
-                    example: false
-                  files:
-                    type: array
-                    items:
-                      type: string
-                      nullable: true
-                example:
-                  config:
-                    type: off
-                  enabled: false
-                  files: []
-        403:
-          $ref: "#/components/responses/Unauthorized"
+
   /api/printerprofiles:
```

Comment:
maybe `printer/profiles` or `printer-profiles`?

Reply:
I designed it so it will stay as much compatible to octoprint as we can https://docs.octoprint.org/en/master/api/printerprofiles.html

Reply:
It was the initial thought if we can create API compatible with octoprint, users can than use third party apps such as https://play.google.com/store/apps/details?id=com.kabacon.octoremote&hl=cs&gl=US with our printers.